modules/styling/spec/cs.spec.tsx

```diff
@@ -22,8 +22,10 @@ import {
   handleCsProp,
   keyframes,
   CSProps,
+  getCache,
 } from '../lib/cs';
 
+const cache = getCache();
```

Comment:
I'm not 100% convinced there should be a `getCache` function if there's already a `getInstance` function. Most of the time, people should not use `getInstance`, but may need to use `getCache`, so I don't know...

Dealing with the cache and the Emotion instance are advance use-cases.